conf/nginx.conf

```diff
@@ -67,7 +70,7 @@ location ^~ __PATH__ {
     fastcgi_param HTTPS on;
     fastcgi_param modHeadersAvailable true;
     fastcgi_param REMOTE_USER $remote_user;
-    fastcgi_pass unix:/var/run/php5-fpm-__NAME__.sock;
+    fastcgi_pass unix:/var/run/php/php7.0-fpm-__NAME__.sock;
```

Comment:
In preparation to Debian Buster (and PHP 7.3),  we could maybe remove the ".0"...?

Reply:
Yep, that's a good idea.
Should be done in the example app first though.